types/layui/README.md

```diff
@@ -0,0 +1,82 @@
+# 使用
```

Comment:
This file shouldn't be here; it will not be published with the package and isn't useful for those attempting to maintain the package.

Reply:
done